source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -268,7 +268,7 @@
                 contentType: 'application/json; charset=utf-8',
                 data: JSON.stringify({ evaluationId: this.id }),
                 success: function (response) {
-                    getDashboardEvaluations(window.ViewData.period);
+                    window.location.reload();
```

Comment:
No, no, maybe something like the user lost something done at client side, but I have no idea.